tutorials/casper/README.md

```diff
@@ -0,0 +1,57 @@
+
+# Casper 
+
+```bash
+module purge
+module load gnu/9.1.0 ncarcompilers openmpi netcdf ncarenv cmake
+```
+
+I also needed a newer version of gmake, it's in /glade/work/jedwards/make-4.3/bin/make
+
+I am using a python environment created with:
+```
+ncar_pylib -c 20201220 /glade/work/$USER/casper_npl_clone
+```
+
+``pip install smartsim``
+
+``smart --device gpu``
```

Comment:
please note that this also could be ``smart --device cpu`` in the case where the user would like to run the database (specifically ML) on cpu nodes

Reply:
My understanding is that --device gpu also builds a cpu version, this seems to be confirmed by testing.